late/scripts/pool.sh

```diff
@@ -67,10 +67,15 @@ if [ ! -f /etc/apt/sources.list.d/dell.list ]; then
     for dir in /cdrom/debs /isodevice/debs /dell/debs;
     do
         if [ -d "$dir" ]; then
-            cd $dir
-            apt-ftparchive packages ../../$dir | sed "s/^Filename:\ ..\//Filename:\ .\//" >> /Packages
+	    cd $dir
+	    apt-ftparchive packages ../../$dir | sed "s/^Filename:\ ..\//Filename:\ .\//" >> /Packages
         fi
     done
+    #Add DHC packages to solve DHC package denpendency issue
+    if [ -d "/cdrom/dhc/packages" ]; then
+        cd /cdrom/dhc/packages
+	apt-ftparchive packages ../../../$dir | sed "s/^Filename:\ ..\//Filename:\ .\//" >> /Packages
```

Comment:
should be `/cdrom/dhc/packages` instead `$dir`

Reply:
correct